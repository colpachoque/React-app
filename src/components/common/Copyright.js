import React from 'react';

const Copyright = () => (<div style={{margin: '10px 0px 0px 20px'}}>netflixroulette</div>)
export default Copyright;