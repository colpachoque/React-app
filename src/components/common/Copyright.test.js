import React from "react";
import Copyright from "./Copyright";
import { shallow, mount } from "enzyme";
import toJson from "enzyme-to-json";
describe("Copyright component", () => {
  it("should be render correctly", () => {
    const component = shallow(<Copyright />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
