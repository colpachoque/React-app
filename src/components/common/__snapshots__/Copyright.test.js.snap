// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Copyright component should be render correctly 1`] = `
<div
  style={
    Object {
      "margin": "10px 0px 0px 20px",
    }
  }
>
  netflixroulette
</div>
`;
